### PR TITLE
add elm block

### PR DESCRIPTION
**Pull request by jeetiss**
implement elm block, but in old Api ;(

### CODE REVIEW

File: packages/elm/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@webpack-blocks/elm",
+  "version": "0.4.0",
+  "description": "Webpack block for elm.",
+  "main": "lib/index",
+  "license": "MIT",
+  "author": "Andy Wermke <andy@dev.next-step-software.com>",
```
Comment (andywer) [inline packages/elm/package.json]: Feel free to enter your name here 😉😄

**Comment by andywer** [comment]
Hey @jeetiss. Thanks for your contribution! :)

I left a comment at the `author` field of the `package.json` 😉
An end-to-end test in `packages/webpack2` testing the elm block in a minimal setup would be more helpful than a unit test, but it's not a major issue. Reminds me of #91...

**Comment by andywer** [comment]
Awesome! 👌

And the end2end test already caught an issue ^^
```
Could not find Elm compiler "elm-make". Is it installed?
```

(See build details. But happens just for node 4, so it might have something to do with npm 2 (nested dependencies) vs npm 3 (flattened deps))

**Comment by andywer** [comment]
One last question: Which old API did you mean by "but in old Api"?

**Comment by jeetiss** [comment]
I can help you rewrite this block in new api #125

**Comment by andywer** [comment]
Ahh, the new webpack-blocks API :)
I fear there are some other to-do tasks on the list before this needs to be done, but thanks!

**Comment by andywer** [comment]
Published as `@webpack-blocks/elm v0.4.0`

**Pull request merged**